drivers/serial/uart_ite_it8xxx2.c

```diff
@@ -28,19 +42,24 @@ enum uart_port_num {
 };
 
 #ifdef CONFIG_PM_DEVICE
-__weak void uart1_wui_isr_late(void) { }
-__weak void uart2_wui_isr_late(void) { }
-
 void uart1_wui_isr(const struct device *gpio, struct gpio_callback *cb,
 		   uint32_t pins)
 {
 	/* Disable interrupts on UART1 RX pin to avoid repeated interrupts. */
 	(void)gpio_pin_interrupt_configure(gpio, (find_msb_set(pins) - 1),
 					   GPIO_INT_DISABLE);
 
-	if (uart1_wui_isr_late) {
-		uart1_wui_isr_late();
-	}
+	/* Refresh console expired time if got UART Rx wake-up event */
+#ifdef CONFIG_UART_CONSOLE_INPUT_EXPIRED
```

Comment:
should this be "if (IS_ENABLED(CONFIG_UART_CONSOLE_INPUT_EXPIRED)) {"?

Reply:
It looks like the symbol `rx_refresh_timeout_work` is only available if `CONFIG_UART_CONSOLE_INPUT_EXPIRED`, so I think the #ifdef is required.

Reply:
Ack.